docs/content/docs/ops/state/state_backends.md

```diff
@@ -71,6 +71,7 @@ This will ensure that the maximum amount of memory is allocated for user code on
 
 The EmbeddedRocksDBStateBackend holds in-flight data in a [RocksDB](http://rocksdb.org) database
 that is (per default) stored in the TaskManager local data directories.
+Unlike storing java objects in HashMapStateBackend, data is stored as serialized byte arrays, which mainly defined by the types serializer, resulting in key comparisons being byte-wise instead of using java's `hashCode()` and `equals()` methods.
```

Comment:
Isn't the verb now missing in this sentence?

```suggestion
Unlike storing java objects in `HashMapStateBackend`, data is stored as serialized byte arrays, which are mainly defined by the type serializer, resulting in key comparisons being byte-wise instead of using Java's `hashCode()` and `equals()` methods.
```